xsos

```diff
@@ -2919,15 +2920,17 @@ SYSCTL() {
   __Pa vm.dirty_background_bytes ""  '{if ($1>0) printf "\"%s\"%s  (%.1f MiB)", $1, H0, $1/1024/1024; else printf "\"0\"%s  (disabled -- check dirty_background_ratio)", H0}'
   __P vm.dirty_expire_centisecs
   __P vm.dirty_writeback_centisecs
+  __P vm.max_map_count ""
+  __P vm.min_free_kbytes ""
```

Comment:
@rhn-jaeshin Heh... I don't particularly care, but what's your rationale for moving `min_free_kbytes` from below to up here?